integration/test/finalizer/finalizer_test.go

```diff
@@ -33,11 +33,7 @@ const (
 )
 
 // Test_Controller_Integration_Finalizer is an integration test to check that
-// multiple controllers do not interfere with finalizer management where they
```

Comment:
This here does not make sense anymore since the controller's informer only watches what you tell it to watch via the label selector. So I dropped a couple of things.